README.md

```diff
@@ -23,13 +23,15 @@ OpenEBS velero-plugin.
 ## Compatibility matrix
 |	OpenEBS/Maya Release	|	Velero Version	|	Velero-plugin Version	|	Codebase	          |
 |	--------------------	|	---------------	|	--------------------	|	------------------- |
-|	0.9                 	|	v0.11.0       	|	0.9.0               	|	v0.9.x	            |
-|	1.0.0               	|	v1.0.0        	|	1.0.0-velero_1.0.0	  |	1.0.0-velero_1.0.0	|
-|	1.1.0               	|	v1.0.0        	|	1.1.0-velero_1.0.0  	|	1.1.0-velero_1.0.0	|
+|	0.9                 	|	>= v0.11.0       	|	0.9.0               	|	v0.9.x	            |
+|	1.0.0               	|	>= v1.0.0        	|	1.0.0-velero_1.0.0	  |	1.0.0-velero_1.0.0	|
```

Comment:
velero-plugin version also can be >=1.0.0-velero_1.0.0 at all the places?
and, we need to mention for other versions of openebs also